src/lib/nginx-rtmp/install

```diff
@@ -21,15 +21,16 @@ STREAMING_SERVER_KEY="${KEY_PATH}/meowl_nginx.key"
 STREAMING_SERVER_CERT="${CERT_PATH}/meowl_nginx.crt"
 STREAMING_SERVER_IP=127.0.0.1
 STREAMING_SERVER_DNS="localhost"
+AUTH_SERVER_URL="https://example.com"
```

Comment:
what is this

Reply:
Authorization server / react express website